npm/test/db/db.test.ts

```diff
@@ -188,7 +195,11 @@ tap.before(async () => {
   for (const idx in dbs) {
     const opts = dbs[idx];
     const db = await DB.new(opts, true);
-    dbObjs[opts.engine! + (opts.type ? opts.type : '')] = db;
+    if (opts.type === 'postgres' && opts['schema'] === non_default_schema) {
+      dbObjs[opts['schema'] + opts.engine! + (opts.type ? opts.type : '')] = db;
```

Comment:
We needed a different DB object for the non default schema, that is why this condition. Previously, the previous db would have been overwritten by the new one.